src/composables/events/FileEvent.js

```diff
@@ -0,0 +1,23 @@
+import { Subject } from 'rxjs';
+
+/**
+ * Represent a rxjs Event object to emit and to receive events about file opening.
+ * The event should be an Object that contains the `id` (path), the `label`
+ * and the `content` of the file.
+ * Example: { id: 'terraform/app.tf', label: 'app.tf', content: 'Hello World' },
+ * @typedef {Subject} OpenFileEvent
+ */
+const OpenFileEvent = new Subject();
+
+/**
+ * Represent a rxjs Event object to emit and to receive events about file selecting.
+ * It is represented by the file's id.
```

Comment:
`The event should be the selected file's id.`

Reply:
done